test/JDBC/input/BABEL-4863-vu-verify.sql

```diff
@@ -0,0 +1,39 @@
+SELECT 123abc;
+GO
+SELECT 0x0o;
+GO
+SELECT 1_2_3;
+GO
+SELECT 0.a;
+GO
+SELECT 0.0a;
+GO
+SELECT .0a;
+GO
+SELECT 0.0e1a;
+GO
+SELECT $1a;
+GO
+SELECT CASE WHEN 1=1  THEN 1 ELSE 0 END[c]
+GO
+SELECT CASE WHEN 1=1  THEN 1 ELSE 0 END'c'
+GO
+SELECT CASE WHEN 1=1  THEN 1 ELSE 0 END"c"
+GO
+select a[c] from babel_4863_t1
+GO
+select a"c" from babel_4863_t1
+GO
+select a'c' from babel_4863_t1
+GO
+declare @v int=1
+select @v[c]
+GO
+declare @v int=1
+select @v'c'
+GO
+declare @v int=1
+select @v"c"
+GO
+select babel_4863_func()[a];
+GO
```

Comment:
Pls add these test cases as well:

- select 1[c], select 1'c', select 1"c"  (i.e. with a numeric constant)
- select 'abc'[def] and variants (currently this works OK, include a test case to make sure it keeps working)
- include cases where such syntax cases occur in a procedure, function and view
- provide upgrade tests for procs/functions/views, for all releases

Reply:
Done, Added Tests and updated upgrade files